locales/en-US.json

```
@@ -38,6 +38,9 @@
     "Import FreeTube subscriptions (.db)": "Import FreeTube subscriptions (.db)",
     "Import NewPipe subscriptions (.json)": "Import NewPipe subscriptions (.json)",
     "Import NewPipe data (.zip)": "Import NewPipe data (.zip)",
+    "To import your subscriptions from YouTube,": "To import your subscriptions from YouTube,",
+    "click here to download your YouTube-Takeout": "click here to download your YouTube-Takeout",
```

```suggestion
    "click here to download your YouTube-Takeout": "click here to download your YouTube Takeout",
```
As per https://en.wikipedia.org/wiki/Google_Takeout